cmd/preview/sleep.go

```diff
@@ -0,0 +1,67 @@
+// Copyright 2023 The Okteto Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package preview
+
+import (
+	"context"
+	"fmt"
+
+	contextCMD "github.com/okteto/okteto/cmd/context"
+	"github.com/okteto/okteto/cmd/utils"
+	oktetoErrors "github.com/okteto/okteto/pkg/errors"
+	oktetoLog "github.com/okteto/okteto/pkg/log"
+	"github.com/okteto/okteto/pkg/okteto"
+	"github.com/spf13/cobra"
+)
+
+// Sleep sleeps a preview
+func Sleep(ctx context.Context) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "sleep <name>",
+		Short: "Sleeps a preview",
+		Args:  utils.ExactArgsAccepted(1, ""),
```

Comment:
```suggestion
		Args:  utils.ExactArgsAccepted(1, ""),
```
We may need to set the preview as default if no argument is passed.

Reply:
Hey @NitishKumar06! The other commands such as `okteto destroy` is also following this practice of accepting one argument atleast and almost, that's why I did the same for `preview` and `sleep`. If we are to switch to a default preview if no arg is provided, then maybe we should implement this logic for other commands as well. What do you think?

Reply:
@NitishKumar06 there is no concept of default preview environment. `<name>` is mandatory in this case

Reply:
got it @pchico83 ! You are correct @rakshitgondwal but it depends on what is the end goal of the command.

For eg. We can pass additional arguments to `sleep` and `wake` command if we don't want to go with the default. However, this may/may not be a perfect fit for a command such as `delete`in which if we don't pass an argument, the command deletes the default namespace. Your thoughts about it @pchico83 ?

Reply:
`okteto ns delete` could default to the current namespace, but `okteto preview destroy` should have the name as argument. Both should ask for confirmation. @rakshitgondwal do you want to create another PR to cover those cases?

Reply:
@pchico83 Here are the changes for `delete` I've implemented:

Issue: #3597
PR: https://github.com/okteto/okteto/pull/3598